tools/add_triage_signature.py

```diff
@@ -612,6 +612,47 @@ def _update_ticket(self, url, issue_key, should_update=False):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class FeatureUsageSignature(Signature):
+    """
+    The signature adds information about features configured in cluster based
+    on feature_usage field. For example: OLM operators, NTP additional
+    source or proxy configuration
+    """
+    OLM_OPERATORS = ["cnv", "lso", "ocs"]
+
+    def __init__(self, jira_client):
+        super().__init__(jira_client, comment_identifying_string="h1. Configured Features")
+
+    @classmethod
+    def isOlm(cls, name):
+        return name.lower() in cls.OLM_OPERATORS
+
+    @classmethod
+    def isGeneralFeature(cls, name):
```

Comment:
rename `name` to `operator_name`

Reply:
No, it should be ``feature_name``

Reply:
Right, my bad. Anyway a later comment of mine says to get rid of this function altogether, because negating is easily done with list comprehension